Moggles/ClientApp/src/menu/AppSelection.vue

```diff
@@ -41,13 +42,17 @@
             });
         },
         methods: {
-            getApplications() {
+            getApplications(applicationName = '') {
                 axios.get('/api/applications')
                     .then((response) => {
                         this.applications = response.data;
-
+                        if (!applicationName == '') {
```

Comment:
use === instead of == both here and below. This will check against type as well

Reply:
Should I remove them from the rest of the file as well?